src/Components/Todo_list/Components/Columns.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import './Todo_list.css'
+import Tasks from './Tasks'
+import Data from '../TodoContext'
+class Columns extends React.Component{
```

Comment:
```suggestion
import Data from '../TodoContext'

class Columns extends React.Component {
```
Hãy tách biệt các phần khác nhau bằng 1 dấu enter
Chú ý soi cho kỹ các dấu cách